### PR TITLE
Update activation-keys.adoc

### PR DESCRIPTION
**Pull request by hustodemon**
Proposing an enhancement of the reactivation keys section (added a howto for salt clients, which was missing).

@Loquacity Please let me know, if the structure/language is ok. Or feel free to change directly. Whatever you prefer :)

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [x] Manager-4.1
- [ ] Manager-4.0

# Links

https://github.com/uyuni-project/uyuni/pull/2947

### CODE REVIEW

**Comment by hustodemon** [comment]
@mcalmer I'd like to draw your eyes on this doc change, since there is no info whatsoever on using reactivation keys on salt (I got everything from reading the code/experimenting). One more thing: IIRC, the reactivation keys for traditionals are deleted after they're used, correct? This seems not to be the case for salt minions (after re-activating, the key is still in `system detail -> reactivation`). Sounds like a bug.

File: modules/client-configuration/pages/activation-keys.adoc [hunk 1]
```diff
@@ -99,6 +99,19 @@ Click btn:[Generate New Key] to create the reactivation key.
...
+----
+grains:
+    susemanager:
+        management_key: "re-1-daf44db90c0853edbb5db03f2b37986e"
+----
+
+You can now restart the [command]``salt-minion`` to apply the reactivation key.
+
+=== Traditional clients
+
```
Comment (Loquacity) [inline modules/client-configuration/pages/activation-keys.adoc:114]: I think the changes are fine, but I would prefer not to use the section headers for such a small section. Can we instead remove the headers, and just treat it as a para? Something like "For Salt clients, after you have created a reactivation key, ... " ?

**Review by Loquacity — CHANGES_REQUESTED** [review]

File: modules/client-configuration/pages/activation-keys.adoc (hunk 1 above)
Comment (hustodemon) [reply modules/client-configuration/pages/activation-keys.adoc]: Sure thing @Loquacity ! I updated that in the latest commit.

**Review by Loquacity — APPROVED** [review]

**Comment by Loquacity** [comment]
Does this need to be backported to 4.1?

**Comment by hustodemon** [comment]
> Does this need to be backported to 4.1?

Yes.

**Comment by hustodemon** [comment]
(shall I open another another PR against https://github.com/uyuni-project/uyuni-docs/tree/manager-4.1?)

**Comment by Loquacity** [comment]
> (shall I open another another PR against https://github.com/uyuni-project/uyuni-docs/tree/manager-4.1?)

I'll do one when I merge this. Sorry for the delay, in a conference right now :joy:

**Comment by hustodemon** [comment]
Awesome, thank you @Loquacity !

**Comment by Loquacity** [comment]
- [ ] Backport to 4.1: https://github.com/uyuni-project/uyuni-docs/pull/723

**Pull request merged**